src/rules/__tests__/new-color-css-vars.test.js

```diff
@@ -0,0 +1,132 @@
+const rule = require('../new-color-css-vars')
+const {RuleTester} = require('eslint')
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true
+    }
+  }
+})
+
+ruleTester.run('no-color-css-vars', rule, {
+  valid: [
+    {
+      code: `{color: 'fg.default'}`
+    },
+    {
+      code: `<circle stroke="var(--color-border-default)" strokeWidth="2" />`
+    },
+    {
+      code: `<circle fill="var(--color-border-default)" strokeWidth="2" />`
+    },
+    {
+      code: `<div style={{ color: 'var(--color-border-default)' }}></div>`
+    },
+    {
+      code: `<Blankslate border></Blankslate>`
+    }
+  ],
+  invalid: [
+    {
+      code: `<Button sx={{color: 'var(--color-fg-muted)'}}>Test</Button>`,
+      output: `<Button sx={{color: 'var(--fgColor-muted, var(--color-fg-muted))'}}>Test</Button>`,
+      errors: [
+        {
+          message: 'Replace var(--color-fg-muted) with var(--fgColor-muted, var(--color-fg-muted))'
```

Comment:
Will folks always need to include a fallback value or is this an interim step as we're moving over to the new system?

Reply:
Interim step! Once v8 is released and fully rolled out, we will refactor this to not include the fallback. We could also add some logic to remove the old fallbacks, but they won't really hurt anything if they stick around.